### DISCUSSION TITLE
When submitting creator and affiliation metadata via the REST API with a valid ORCID or ROR ID, fields like creatorName should be populated dynamically from metadata in ORCID or ROR

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#90] [General Suggestions]
### What is the problem that your suggestion solves?

When submitting creator and affiliation metadata via the REST API with a valid ORCID or ROR ID, users still need to set the rest of the metadata for that entity manually. 

### What solution might meet your needs?

Similar to the Fabrica form, fields like givenName, familyName, and creatorName should be populated dynamically from metadata in ORCID or ROR corresponding to the ID.

### What alternatives have you tried or considered?

Manually setting the entity's other metadata properties.

### Is there anything else you would like to share?

https://app.frontapp.com/open/cnv_b732k45?key=jb0U-OIPlAKQCXJC0d_yaCi340j-xHTL

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
This could also be done for funder RORs.
